### PR TITLE
Private/timar/configdefaults

### PR DESCRIPTION
**Pull request by timar**
* Resolves: #10049 
* Target version: master 

### Summary
Synced default values of configuration settings between code and coolwsd.xml.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1505,7 +1505,10 @@ AS_IF([test `uname -s` = "FreeBSD"],
 AS_IF([test -n "$with_compiler_plugins"],
       [CPPFLAGS="$CPPFLAGS -Xclang -load -Xclang ${with_compiler_plugins}/compilerplugins/obj/plugin.so -Xclang -add-plugin -Xclang loplugin -Xclang -plugin-arg-loplugin -Xclang --cool-base-path=\${abs_top_srcdir}"])
 
-COOLWSD_CONFIGDIR=${sysconfdir}/${PACKAGE}
+if test "x${prefix}" = "xNONE"; then
+    prefix=/usr/local
+fi
+eval COOLWSD_CONFIGDIR="${sysconfdir}/${PACKAGE}"
```
Comment (vmiklos) [inline configure.ac:1511]: Nit: why do you need eval here? I would expect sysconfdir is /etc and PACKAGE is coolwsd at this stage.

**Review by vmiklos — APPROVED** [review]
- OK, I see that coolwsd.xml already uses @COOLWSD_LOGLEVEL@, so this sets the built-in default to the same value, makes sense to me.
- Same for @NUM_PRESPAWN_CHILDREN@, which is now used at more places, OK.
- cleanup: again this was true at a config level, the rest is now adjusted to match, good.
- `out_of_focus_timeout_secs`: same here, good.
- SSL termination: yes, I remember that story, good to have false everywhere.
- configdir: makes sense, hardcoding /etc was not nice.

All in all: looks good to me, thanks.

File: configure.ac (hunk 1 above)
Comment (timar) [reply configure.ac:1511]: Without eval the value was literal `${prefix}/etc/coolwsd` when I did not specify sysconfdir separately. The reason is that these paths are evaluated later. Autoconf sets it so, that prefix etc. can be changed even as late as `make install`.

Comment (vmiklos) [reply configure.ac:1511]: Ah I see, thanks.

**Pull request merged**